excellence-fee/admin/admin_input.php

```diff
@@ -0,0 +1,52 @@
+<?php
+	// $insert_path=plugin_dir_path( __FILE__ ) . "insert_fee.php";
+	$insert_path=plugins_url( 'insert_fee.php', __FILE__ )
+?>
+
+<form method="post" action="<?php echo $insert_path; ?>" name="fee_form" class="fee_form">
+	<div class="input-form">
+		<div class="">Enter the Fee Amount: </div>
+		<div class="input_fee">
+			<input type="text" name="excel_fee" class="excel_fee" placeholder="enter excellence fee">
+		</div>
+		<input type="submit" name="excel_fee_submit" class="excel_fee_submit" value="SAVE">
+	</div>
+</form>
+<script
```

Comment:
@Shraddha1596 same mistake done in OMAR PROJECT AND DOING SAME HERE, WHY INCLUDING JQUERY AGAIN N AGAIN, NOT LEARNING FOR PAST MISTAKES